keycloak-phone-provider/src/main/java/cc/coopersoft/keycloak/phone/providers/spi/impl/TokenCodeServiceImpl.java

```diff
@@ -87,7 +87,7 @@ public boolean isAbusing(String phoneNumber, TokenCodeType tokenCodeType) {
                 .setParameter("type", tokenCodeType.name())
                 .getResultList();
 
-        return entities.size() > 3;
+        return entities.size() > 5;
```

Comment:
@towe75 Please note: This limits the amound of verification codes being generated for one phone number within an hour. After hitting this limit there is no way to reset or increase this by now - we might want to think about that.

Reply:
@dominikschopf : a parameter makes sense. I propose skip this fraud check if the new parameter has a value of 0. This helps us to do load/debug tests etc.